tests/integration/components/mutually_exclusive/test_checkbox_single_checkbox_override.py

```diff
@@ -7,7 +7,7 @@ class TestCheckboxSingleCheckboxOverride(IntegrationTestCase):
     with single checkbox override function as expected. These tests emulate the non-JS version.
     """
 
-    def setUp(self):
+    def setUp(self, setting_overrides=None):
```

Comment:
I can't see why the new argument is needed?

Reply:
pylint moans

Reply:
about what?

Reply:
```tests/integration/components/mutually_exclusive/test_currency_single_checkbox_override.py:11:4: W0221: Parameters differ from overridden 'setUp' method (arguments-differ)```

Reply:
If we're adding an argument for it, then it should be passed to the super().